core/amber/src/main/scala/edu/uci/ics/texera/web/resource/GoogleResource.java

```diff
@@ -29,17 +25,14 @@
 public class GoogleResource {
     private static final int GOOGLE_TIMEOUT_IN_MS = 10000;
     private static final String APPLICATION_NAME = "Texera";
-    private static final Config config = ConfigFactory.load("google_api");
-    private static final String TOKENS_DIRECTORY_PATH = Utils.amberHomePath()
-            .resolve("../conf").resolve(config.getString("google.tokenPath")).toString();
+    private static final String TOKENS_DIRECTORY_PATH = "tokens";
     private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
 
     /**
      * Global instance of the scopes required by this quickstart.
      * If modifying these scopes, delete your previously saved tokens/ folder.
      */
-    private static final String CREDENTIALS_FILE_PATH = Utils.amberHomePath()
-            .resolve("../conf").resolve(config.getString("google.credentialPath")).toString();
+    private static final String CREDENTIALS_FILE_PATH = "/credentials.json";
```

Comment:
ditto. also why does this one has `/` hardcoded instead of using `path.resolve`? (it might not work for windows)?

Reply:
fixed.